services/ScheduleCreate.proto

```diff
@@ -26,28 +26,46 @@ option java_package = "com.hederahashgraph.api.proto.java";
+and not the node and network fees. If the optional <tt>payerAccountID</tt> is set, the network charges this account. Otherwise it charges the 
+payer of the originating <tt>ScheduleCreate</tt>.  
+
+Two <tt>ScheduleCreate</tt> transactions are <i>identical</i> if they are equal in all their fields other than <tt>payerAccountID</tt>. 
+(Here "equal" should be understood in the sense of gRPC object equality in the network software runtime. In particular, a gRPC object with 
+<a href="https://developers.google.com/protocol-buffers/docs/proto3#unknowns">unknown fields</a> is not equal to a gRPC object without
+unknown fields, even if they agree on all known fields.) 
+
+A <tt>ScheduleCreate</tt> transaction that attempts to re-create an identical schedule already in state will receive a receipt with status 
+<tt>IDENTICAL_SCHEDULE_ALREADY_CREATED</tt>; the receipt will include the <tt>ScheduleID</tt> of the extant schedule, which may be 
+used in a subsequent <tt>ScheduleSign</tt> transaction.
+
+Other notable response codes include,
+<ul>
+  <li> <tt>INVALID_ACCOUNT_ID</tt> if the schedule's designated payer account does not exist or was deleted.
+  <li> <tt>UNSCHEDULABLE_TRANSACTION</tt> if the scheduled transaction is not in the scheduling whitelist.
+  <li> <tt>UNRESOLVABLE_REQUIRED_SIGNERS</tt> if the scheduled transaction refers to entities that do not exist or have been deleted.
+  <li> <tt>INVALID_SIGNATURE</tt> if the <tt>ScheduleCreate</tt> signature map included invalid signatures. (No schedule is created in this case.)
+</ul>
 */
+
 message ScheduleCreateTransactionBody {
-  bytes transactionBody = 1; // The transaction serialized into bytes that must be signed
-  Key adminKey = 2; // (optional) The Key which is able to delete the Scheduled Transaction (if tx is not already executed)
-  AccountID payerAccountID = 3; // (optional) The account which is going to pay for the execution of the Scheduled TX. If not populated, the scheduling account is charged
-  SignatureMap sigMap = 4; // (optional) Signatures that could be provided (similarly to how signatures are provided in ScheduleSign operation) on Scheduled Transaction creation
-  string memo = 5; // (optional) Publicly visible information about the Scheduled entity, up to 100 bytes. No guarantee of uniqueness.
+  SchedulableTransactionBody scheduledTransactionBody = 1; // The scheduled transaction
+  string memo = 2; // An optional memo with a UTF-8 encoding of no more than 100 bytes which does not contain the zero byte
+  Key adminKey = 3; // An optional Hedera key which can be used to sign a ScheduleDelete and remove the schedule
+  AccountID payerAccountID = 4; // An optional id of the account to be charged the service fee for the scheduled transaction at the consensus time that it executes (i    f ever); defaults to the ScheduleCreate payer if not given
```

Comment:
Lots of space between the i and f of `if`

Reply:
Fixed.